docs/how-to/deploy-omni-view-on-render.md

```
@@ -104,15 +104,23 @@ If creating the service manually in the Render Dashboard:
 
 ## 🔧 Troubleshooting Render Deploys
 
-### 1. Error: `bash: line 1: uvicorn: command not found` (Exit Status 127)
+### 1. Error: `ModuleNotFoundError: No module named 'src'` (Exit Status 1) or `uvicorn: command not found` (Exit Status 127)
 
 - **Symptom:** Deployment log fails during the start/deploy phase with:
+  ```text
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)